Utils.py

```diff
@@ -739,6 +739,22 @@ def is_kivy_running():
         root.update()
 
 
+def ask_yes_no_cancel(title: str, text: str) -> Optional[bool]:
+    """
+    Wrapper for tkinter.messagebox.askyesnocancel, that creates a popup dialog box with yes, no, and cancel buttons.
+
+    :param title: Title to be displayed at the top of the message box.
+    :param text: Text to be displayed inside the message box.
+    :return: Returns True if yes, False if no, None if cancel.
+    """
+    from tkinter import Tk, messagebox
```

Comment:
I don't know if there is a particular reason to put this import here instead of at the beginning of the file. Because PEP 8 says the it should be put at the top of the file: https://peps.python.org/pep-0008/#imports

Reply:
tkinter may not exist on the system. this method should really be preferring kivy, but i couldn't get a popup box to work in kivy at the time and it's kind of out of scope for this PR anyways; this is just moving my own custom implementation into utils for general use. the local import is purposeful and correct here though.